### ISSUE TITLE
kendoPDFExportComponent, Bug: the image doesn't show in the exported pdf output for the first time

### ISSUE THREAD

**Issue by Levon770** [telerik/kendo-angular#3976]
**Describe the bug**
Using the simple example from docs, with current versions (Angular 13), the image should be shown in the pdf output but it does not. But for the second time exporting the pdf, the image is shown in the pdf output.

The list of installed packages from KENDO:

```
    "@progress/kendo-angular-buttons": "latest",
    "@progress/kendo-angular-common": "latest",
    "@progress/kendo-angular-dateinputs": "latest",
    "@progress/kendo-angular-dialog": "latest",
    "@progress/kendo-angular-dropdowns": "latest",
    "@progress/kendo-angular-excel-export": "latest",
    "@progress/kendo-angular-grid": "latest",
    "@progress/kendo-angular-icons": "latest",
    "@progress/kendo-angular-inputs": "latest",
    "@progress/kendo-angular-intl": "latest",
    "@progress/kendo-angular-l10n": "latest",
    "@progress/kendo-angular-label": "latest",
    "@progress/kendo-angular-navigation": "latest",
    "@progress/kendo-angular-pdf-export": "latest",
    "@progress/kendo-angular-popup": "latest",
    "@progress/kendo-angular-progressbar": "latest",
    "@progress/kendo-angular-treeview": "latest",
    "@progress/kendo-angular-upload": "latest",
    "@progress/kendo-data-query": "latest",
    "@progress/kendo-drawing": "latest",
    "@progress/kendo-licensing": "latest",
    "@progress/kendo-svg-icons": "latest",
```

**To Reproduce**
Look at this, to reproduce:

[https://stackblitz.com/edit/angular-itjvbe-wyn6ep](https://stackblitz.com/edit/angular-itjvbe-wyn6ep)

**Comment by mbechev** [comment]
To add some more context around this case, the issue is caused after setting a fixed **paperSize**. If the option is omitted or set to 'auto' the image is visible.
https://stackblitz.com/edit/angular-itjvbe-xb4lid

**Comment by Levon770** [comment]
@mbechev I need to set paper size as letter. So will wait on bugfix on one of updated versions.
Is there any news on when I will get update?

**Comment by erothfield** [comment]
I just came across this as well, when I noticed I get the image in the pdf on _alternating_ downloads!
In my case, using the forcePageBreak attribute is the culprit.

**Comment by erothfield** [comment]
In my project, we had not upgraded to latest kendo for angular.  I found if we update to latest (kendo-angular -> 12.1, kendo-drawing -> 1.17.5), then _downgrade_ the drawing package to 1.17.3, we could get around the problem.

**Comment by MellowYeller** [comment]
I noticed in our case the PDF is generated properly in Firefox, but not Chrome. paperSize="A4" caused the image to not load reliably in Chrome for us.

Firefox 115.0.2
Chrome 115.0.5790.110

Downgrading kend-drawing to 1.17.3 (from 1.17.5) seems to fix the issue, as @erothfield mentioned.

**Comment by tsvetomir** [comment]
Thanks everyone for taking the time to investigate this issue. The latest development version (`v1.17.6-dev.202308221133`) should fix it. Please, give it a try and let us know if it works for you:

`npm install --save @progress/kendo-drawing@dev`

**Comment by dtopuzov** [comment]
@tsvetomir `@progress/kendo-drawing@dev` looks good, let's ship it.

**Comment by tsvetomir** [comment]
Fixed in `@progress/kendo-drawing` v1.17.6
